Source/ProgressionSystemRuntime/ProgressionSystemRuntime.Build.cs

```diff
@@ -23,7 +23,7 @@ public ProgressionSystemRuntime(ReadOnlyTargetRules Target) : base(Target)
 				// Bomber modules 
 				"MyUtils" 
 				, "PoolManager" // Star and Widget Actors
-				, "MetaCheatManager" // PSCheatExtension
+				, "MetaCheatManager", "NewMainMenu",  // PSCheatExtension
```

Comment:
Modular Game Feature modules don't allow to be referenced from outside by anything else, even by other similar modules.

Instead, please sync to latest Bomber's Develop [21e12aa](https://github.com/JanSeliv/Bomber/commit/21e12aa17109fc2fdebf10db8a57e3f65b373b31), and bind to `OnAnyCinematicStarted` delegate from `UMyBlueprintFunctionLibrary::GetLocalPlayerController()`